dev-tools/jenkins_ci.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euox pipefail
+
+: "${HOME:?Need to set HOME to a non-empty value.}"
+: "${WORKSPACE:?Need to set WORKSPACE to a non-empty value.}"
+: "${beat:?Need to set beat to a non-empty value.}"
+
+if [ ! -d "$beat" ]; then
+  echo "$beat does not exist"
+  mkdir -p build
+  touch build/TEST-empty.xml
+  exit
+fi
+
+# Setup Go.
+export GOPATH=${WORKSPACE}
+export PATH=${GOPATH}/bin:${PATH}
+if [ -f ".go-version" ]; then
+  eval "$(gvm $(cat .go-version))"
+else
+  eval "$(gvm 1.7.5)"
```

Comment:
We should probably update this to 1.8

Reply:
We could probably remove that these scripts. I had it there for the time when we didn't have .go-version in all branches. But _I think_ we do now.